po/es.po

```diff
@@ -1514,24 +1505,23 @@ msgstr "Descargar"
 
 #: src/pbview.cpp:368
 msgid "Delete"
-msgstr "Eliminar"
+msgstr "Borrar"
 
 #: src/pbview.cpp:369
 msgid "Purge Finished"
-msgstr "Purgar finalizadas"
+msgstr "Limpiar las finalizadas"
 
 #: src/pbview.cpp:370
 msgid "Toggle Automatic Download"
-msgstr "Cambiar a descarga automática"
+msgstr "Conmutar la descarga automática"
 
 #: src/pbview.cpp:371
 msgid "Play"
 msgstr "Reproducir"
 
 #: src/pbview.cpp:372
-#, fuzzy
 msgid "Mark as Finished"
-msgstr "Purgar finalizadas"
+msgstr "Marcar como terminada/s"
```

Comment:
The slash looks suspicious to me, is this correct?

(I don't speak Spanish, feel free to ignore)